src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/CompilerPass/RegisterDatePointTypePassTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Tests\DependencyInjection\CompilerPass;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Doctrine\DependencyInjection\CompilerPass\RegisterDatePointTypePass;
+use Symfony\Bridge\Doctrine\Types\DatePointType;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class RegisterDatePointTypePassTest extends TestCase
+{
+    public function testRegistered()
+    {
+        $container = new ContainerBuilder();
```

Comment:
we need to skip the test if the `DatePoint` class does not exist (see the low deps failure)

Reply:
skip added